docs/manage_device.rst

```diff
@@ -49,4 +49,28 @@ To grant **Super user** permissions to another user, follow these steps.
 The users who have been granted the **Super user** permissions will have a yellow key indicator in front of their name, and will be able to see the **Device** dashboard with both the **Content** and **Permissions** tabs.
 
 	.. image:: img/permissions-keys.png
-	  :alt: permissions indicators
\ No newline at end of file
+	  :alt: permissions indicators
+
+
+.. _device_info:
+
+
+View Device Info
+----------------
+
+To view the detailed info of the device where Kolibri is running on, use the **Info** tab in the  **Device** dashboard (|info| icon). This information will be useful in case you need to report an issue with Kolibri to FLE support team. Make note or copy the following device details:
+
+* Kolibri version
+* Server IP/URL(s)
+* Database path
+* Device name
+* Operating system 
+* Free disk space
+* Server time
+* Server timezone
+
+
+.. figure:: img/device-info.png
```

Comment:
![image](https://user-images.githubusercontent.com/2367265/33603121-60fb64b6-d966-11e7-93ae-4d0998ea5dc8.png)

Could be worth photoshoping this to say 0.7.0 and maybe a fake IP

Reply:
This is probably fine, the page will change anyways, so at some point when things are more stable, we can work with these screen shots.

I think another interesting discussion is whether this version of the user docs is actually the source data (text-wise) for a different, illustrated version.